snakesist/exist_client.py

```diff
@@ -138,9 +191,13 @@ def __init__(
     def _join_paths(*args):
         return "/".join(s.strip("/") for s in args)
 
-    def _get_request(self, url: str, query: Optional[str] = None) -> bytes:
+    def _get_request(self, url: str, query: Optional[str] = None, wrap: bool = True) -> bytes:
+        if wrap:
+            wrap_value = "yes"
+        else:
+            wrap_value = "no"
         if query:
-            params = {"_howmany": "0", "_indent": "no", "_query": query}
+            params = {"_howmany": "0", "_indent": "no", "_wrap": wrap_value, "_query": query}
```

Comment:
one *could* do this: `"_wrap": ("no", "yes")[wrap]`, but admittedly it's rather cryptic.

or: `"_wrap": "yes" if wrap else "no"`

in order to not allocate the `wrap` variable.

Reply:
i did the second one, it's much more readable. 
& learned something new, i wasn't aware of this syntax feature :)